packages/code_excerpt_updater/test_data/src/dartdoc.md

```diff
@@ -0,0 +1,8 @@
+## API docs
+
+Markdown code block opened and closed with more than 3 backticks
+that contains another Markdown code block declared with just 3.
```

Comment:
... except that it doesn't contain another Markdown code block if I'm reading the test correctly ;-)